<launch>
    <param name="use_sim_time" value="true"/>
    <include file="$(find babybuggy_config)/launch/babybuggy_config.launch"/>
    <!-- <include file="$(find babybuggy_config)/launch/babybuggy_gmapping.launch"/> -->
    <!-- <include file="$(find babybuggy_config)/launch/babybuggy_visualize.launch"/> -->

    <node name="odometry_tf_converter" pkg="odometry_tf_converter" type="odometry_tf_converter_node" output="screen">
        <param name="~odom_sub_name" type="string" value="naive_odom"/>
    </node>
</launch>
